pages/validator/external-chains/overview.md

```diff
@@ -34,7 +34,7 @@ See below for details. Read this entire article before you begin supporting exte
   - [Moonbeam](./moonbeam)
   - [Polygon](./polygon)
   - [Celo](./celo)
-  - [Arbitrum](./arbitrum)
+  - [Optimism](./optimism)
```

Comment:
This is replacing `Arbitrum` link, perhaps we should add `Optimism` as a new line item

Reply:
ahh thanks for flagging it, didn't mean to replace arbritrum, will fix it

Reply:
Fixed